Formula/solr.rb

```diff
@@ -46,7 +46,7 @@ def plist
             <string>start</string>
             <string>-f</string>
             <string>-s</string>
-            <string>/usr/local/var/lib/solr</string>
+            <string>#{var/"lib/solr"}</string>
```

Comment:
This is more in line with what we do elsewhere.
```suggestion
            <string>#{HOMEBREW_PREFIX}/var/lib/solr</string>
```

Reply:
`var` is `HOMEBREW_PREFIX/"var"` by definition, so this could actually just be
```ruby
#{var}/"lib/solr"
```

Reply:
I aimed for consistency within this source code file. There is a definition above which uses `#{var/"lib/solr"}` that's all...